tasks/main.yml

```
@@ -2,48 +2,53 @@
 # tasks file for roles/upgrade-distpackages
 
 - name: system packages | upgrade
-  become: yes
+  become: true
   yum:
     name: "{{ item }}"
     exclude: "{{ upgrade_distpackages_excludes | join(',') }}"
     state: latest
-    update_cache: yes
+    update_cache: true
   with_items: "{{ upgrade_distpackages }}"
   # Ignore ansible-lint warning about "latest"
   tags:
-  - skip_ansible_lint
+    - skip_ansible_lint
 
 - name: system | current running kernel
   command: uname -r
   register: running_kernel_version
-  check_mode: no
-  changed_when: False
+  check_mode: false
+  changed_when: false
 
 # http://serverfault.com/a/601432
 - name: system | latest installed kernel
-  shell: rpm -q kernel --qf '%{BUILDTIME} %{VERSION}-%{RELEASE}.%{ARCH}\n' | tail -n 1 | cut -f 2 -d ' '
+  shell: >-
+    rpm -q kernel --qf '%{BUILDTIME} %{VERSION}-%{RELEASE}.%{ARCH}\n' |
+    tail -n 1 | cut -f 2 -d ' '
   register: latest_kernel_version
-  check_mode: no
-  changed_when: False
+  check_mode: false
+  changed_when: false
   # Ignore ansible-lint warning about using rpm (this is just a query)
   tags:
-  - skip_ansible_lint
+    - skip_ansible_lint
 
 - name: system | check if reboot needed
   set_fact:
-    system_reboot_needed: "{{ running_kernel_version.stdout != latest_kernel_version.stdout }}"
+    system_reboot_needed: >-
+     "{{ running_kernel_version.stdout != latest_kernel_version.stdout }}"
 
 - debug:
-    msg: "Reboot needed (kernel): Current:{{ running_kernel_version.stdout }} Latest:{{ latest_kernel_version.stdout }}"
+    msg: >-
+      "Reboot needed (kernel): Current:{{ running_kernel_version.stdout }}"
+      " Latest:{{ latest_kernel_version.stdout }}"
```

May need to adjust the leading space on this
```suggestion
       Latest:{{ latest_kernel_version.stdout }}
```